man/docker-cp.1.md

```diff
@@ -8,6 +8,9 @@ docker-cp - Copy files/folders between a container and the local filesystem.
 **docker cp**
 [**--help**]
 CONTAINER:PATH LOCALPATH|-
+
+**docker cp**
+[**--help**]
```

Comment:
Just wondering if this is possibly confusing because we're showing the same command twice here